packages/cli/src/api/upload.js

```diff
@@ -1,4 +1,5 @@
 const axios = require('axios');
+const { version } = require('../../package.json');
```

Comment:
Will lerna actually change the version?

Reply:
Yes, learn will change the version and on npm publish, the GitHub action will publish the correct version.